### DISCUSSION TITLE
iPXE native driver support for Realtek RTL8125B series NIC

### DISCUSSION THREAD

**Discussion by kevinr1333** [ipxe/ipxe#1120] [General]
Hello,

Is there any plan to support Realtek 8125B series NIC? Thanks.

BRs
Kevin

**Comment by NiKiZe** [comment]
Searching other posts I find #540

Please note, iPXE itself has no "plan" to add native drivers for any NIC, most of them work fine in SNP/UNDI mode.
If you however have a need for native driver you can reach out for commercial support. But hardware for testing as well as hardware manuals (the details about the hardware and which register does what etc, will be needed)

**Reply by amdhugo** [reply]
Is there a reference on how reaching out for commercial support from you?
BTW, Is it possible to request to add any drivers to snp mode if we hit issues also on that?

**Reply by NiKiZe** [reply]
See ipxe.org.

SNP is an interface provided by EFI firmware, nothing needs to be added to iPXE for that, but your NIC will need to have a proper working efi ROM.
